### ISSUE TITLE
setup instructions for Angular need update for NS 2.3/Angular 2.0

### ISSUE THREAD

**Issue by andy-tai** [rborn/nativescript-i18n#12]
The instructions on how to set up nativesceript-i18n with Angular by import I18N_PROVIDER in main.ts 

```
import {I18N_PROVIDERS} from "nativescript-i18n/angular";

nativeScriptBootstrap(AppComponent, [I18N_PROVIDERS]);
```

does not seem to work with NativeScript 2.3 with Angular 2.0 (official release) any more... needs updating

**Comment by andy-tai** [comment]
Hi, I find that with some changes to angular.ts in nativescript-i18n, 

[patch.txt](https://github.com/rborn/nativescript-i18n/files/492215/patch.txt)

then in the application's code, do not do the changes for main.ts as currently suggested,, but in the app's app.module.ts file,  but do these:

```
import { L } from 'nativescript-i18n/angular';
...

@NgModule ({
....
    declarations :  [
        AppComponent,
        L , 
      ... 
   ], 
.... 
})
export class AppModule {}

```

and then other code using i18n can work without changes.

**Comment by rborn** [comment]
Hi @andy-tai could you please create a PR for it? 😄 
Thank you.

**Comment by jamlfy** [comment]
@rborn is simple! :D

**Comment by rborn** [comment]
@alejonext of course, but a PR would be even easier :) and would also be part of the contribution from the community 😉

**Comment by andy-tai** [comment]
@rborn  will do

**Comment by rborn** [comment]
Closing, as we merged #15
